astrobee/config/communications/ground_dds_ros_bridge.config

```diff
@@ -25,7 +25,7 @@
 -- Robot we are receiving data from
 connecting_robot = "Bumble"
 
--- Domain id, needs to be 23 for space
+-- Domain id
```

Comment:
Maybe you thought it would be better to sanitize this for public release? Too bad, because it seems like useful info. Maybe it could be rewritten as:
```
-- NASA-internal users: For guidance on domains check [wiki URL]
```

Reply:
Do you still have that URL? I no longer have it. Also, domain id is discussed in each activity's TRR so I'm not sure if a comment is needed.